src/Maker/MakeTwigExtension.php

```diff
@@ -83,5 +107,9 @@ public function configureDependencies(DependencyBuilder $dependencies): void
             AbstractExtension::class,
             'twig'
         );
+        $dependencies->addClassDependency(
+            RuntimeExtensionInterface::class,
+            'twig'
+        );
```

Comment:
Not needed :). We just need to mention one class here (e.g. `AbstractExtension`) as a way to make sure that `twig` itself is installed.